src/schemas/ev42/ev42_rw.cxx

```diff
@@ -4,7 +4,7 @@
 #include "../../HDFWriterModule.h"
 #include "../../h5.h"
 #include "../../helper.h"
-#include "schemas/ev42_events_generated.h"
+#include "ev42_events_generated.h"
```

Comment:
It would be preferable to keep the `schemas/` prefix here, can that be accomplished using a Conan-provided `streaming-data-types`?